spec/requests/api/v1/passwords/update_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe 'PATCH /api/v1/users/password', type: :request do
+  let(:user) { create(:user) }
+  let(:password_token) { user.send(:set_reset_password_token) }
+  let!(:headers) do
+    params = {
+      reset_password_token: password_token,
+      redirect_url: ENV['PASSWORD_RESET_URL']
+    }
+    get edit_user_password_path, params: params, headers: auth_headers
```

Comment:
Maybe this `get` and the `edit_response_params` shouldn't be inside this `let`. 

Can you try moving them to a `before` action? or maybe inside each test itself.